scripts/testCliWorkflow.js

```diff
@@ -0,0 +1,221 @@
+        v.steps[i] = {
+          ...v.steps[i],
+          run: `mkdir -p ${artPath} && tar -c${verbose ? 'v' : ''}f ${artPath}/${c.with.name} ${c.with.path.split('\n').join(' ')}`
+        };
+        delete v.steps[i].uses;
+        delete v.steps[i].with;
+      } else
+      // Replace Download Artifacts
+      if (c.uses?.indexOf("actions/download-artifact") >= 0) {
+        // TODO: Research support for wildcards
+        v.steps[i] = {
+          ...v.steps[i],
+          run: `tar -x${verbose ? 'v' : ''}f ${artPath}/${c.with.name}`
+        };
+        delete v.steps[i].uses;
+        delete v.steps[i].with;
+        } else {
+        // ID specific actions
+        switch (c.id) {
+          case "install-rust": {
+            v.steps[i] = {
+              ...v.steps[i],
+              // run: "yum install cargo -y\ncargo --version\n"
+              run: "apt update -y\napt install build-essential curl -y\ncurl https://sh.rustup.rs -sSf > __install_rust.sh\nchmod u+x __install_rust.sh\n./__install_rust.sh -y\nsource $HOME/.cargo/env\ncargo --version\n"
+            };
+            delete v.steps[i].uses;
+            delete v.steps[i].with;
+            break;
+          }
+          case "build-binary": {
+            v.steps[i] = {
+              ...v.steps[i],
+              // run: "yum install cargo -y\ncargo --version\n"
+              run: "source $HOME/.cargo/env\ncargo --version\n" + c.run
+            };
+            break;
+          }
+        }
+      }
+    }
+  }
+
+  // Output generated workflow
+  const genWfPath = path.resolve(__dirname, "../.github", `_act_${path.basename(wfPath)}`);
+  console.log("Generating new workflow...");
+  await _sleep();
+  _handle(() => {
+    fs.writeFileSync(genWfPath, yaml.dump(wf, { indent: 2, lineWidth: -1 }), { flag: "w" });
+  }, `Unable to write "${genWfPath}"`);
+
+  if (process.argv.indexOf(opts.dryRun[0]) > 0 || process.argv.indexOf(opts.dryRun[1]) > 0) {
+    console.log("New workflow saved:", genWfPath);
+  } else {
+    // Execute workflow locally
+    console.log("Executing new workflow...");
+    await _sleep();
+    const runAct = cp.spawn("act", [`-r${verbose ? 'v' : ''}W`, genWfPath, "-e", epPath], { stdio: "inherit" });
+
+    runAct.on("close", async () => {
+      console.log("Copying existing artifacts...");
+      await _sleep();
+      const testPath = path.resolve(__dirname, "..", "__tests__", "__results__", "nektos_act");
+      _handle(() => {
+        const copyPath = JSON.parse(cp.execSync("docker inspect act-toolcache").toString().trim())[0].Mountpoint;
+        fs.mkdirSync(path.join(copyPath, "artifacts"), { recursive: true });
+        fs.mkdirSync(testPath, { recursive: true });
+        cp.execSync(`tar -zc${verbose ? 'v' : ''}f __act__artifacts.tgz -C ${copyPath} .`);
+        cp.execSync(`tar -zx${verbose ? 'v' : ''}f __act__artifacts.tgz -C ${testPath}`);
+        fs.rmSync("__act__artifacts.tgz", { force: true });
```

Comment:
Would the following work instead?
`fs.unlinkSync("__act__artifacts.tgz");`

Then the note below about Node 14.14.0 may be unnecessary 🙂 

Sorry I didn't think about this when we discussed it before 😢